options/locale/locale_en-US.ini

```diff
@@ -1372,6 +1372,19 @@ settings.transfer_desc = Transfer this repository to a user or to an organizatio
 settings.transfer_notices_1 = - You will lose access to the repository if you transfer it to an individual user.
 settings.transfer_notices_2 = - You will keep access to the repository if you transfer it to an organization that you (co-)own.
 settings.transfer_form_title = Enter the repository name as confirmation:
+settings.signing_settings = Signing Verification Settings
+settings.trust_model = Signature Trust Model
+settings.trust_model.default = Default Trust Model
+settings.trust_model.default.desc= Use the default repository trust model for this installation.
+settings.trust_model.collaborator = Collaborator
+settings.trust_model.collaborator.long = Collaborator: Trust signatures signed by keys of collaborators
+settings.trust_model.collaborator.desc = Signatures from keys owned by collaborators of this repository will be trusted - (whether they match the Committer or not). Otherwise, commits will be marked untrusted if the signature matches the committer and unmatched if the signature otherwise verifies.
```

Comment:
```suggestion
settings.trust_model.collaborator.desc = Signatures from collaborators of this repository will be trusted (whether they match the committer or not), Otherwise they will be marked as unmatched if signature is valid or untrusted if additionally the signature matches the committer.
```